crates/whiskers/src/runner/mod.rs

```diff
@@ -17,6 +18,7 @@ use std::sync::Arc;
 use crate::Sketch;
 pub use animation::AnimationOptions;
 use convert_case::Casing;
+pub use debug::DebugOptions;
```

Comment:
Not `pub`

Reply:
here it's `pub` because it's used by the context. but I removed it from `lib.rs` and `prelude.rs`. is it ok?